voluncheer/opportunityboard/tests_views.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+from django.urls import reverse
+
+
+class OpportunityboardTestCase(TestCase):
+    """Test cases for Opportunityboard view"""
+
+    def test_opportunityboard_page_loads(self):
+        """Tests opportunityboard page loads"""
+        response = self.client.get(reverse("opportunityboard"))
+        self.assertEqual(response.status_code, 200)
+
+    def test_select_page_loads(self):
+        """Tests select page loads"""
+        response = self.client.get(reverse("select"))
+        self.assertEqual(response.status_code, 200)
+
+    def test_post_an_opportunity_page_loads(self):
+        """Tests post_an_opportunity page loads"""
+        response = self.client.get(reverse("post_an_opportunity"))
+        self.assertIn(response.status_code, [200, 302])
```

Comment:
Why are we allowing for success or redirect? Same for below.

Reply:
302 error does not mean webpage couldn’t be loaded. It's like too many redirections or so... Hence I omit it